.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2.0
+
+# Default actions to perform on each Emacs version
+default: &default-steps
+  steps:
+    - checkout
+    - run: apt-get update && apt-get install make
+    - run: make elpa
+    - run: emacs --version
+    - run: make test
+    - run: make test-bytecomp
+
+# Enumerated list of Emacs versions
+jobs:
+  test-emacs-25:
+    docker:
+      - image: silex/emacs:25-dev
+        entrypoint: bash
+    <<: *default-steps
+
+  test-emacs-26:
+    docker:
+      - image: silex/emacs:26-dev
+        entrypoint: bash
+    <<: *default-steps
+
+  test-emacs-master:
+    docker:
+      - image: silex/emacs:master-dev
+        entrypoint: bash
+    <<: *default-steps
+#
```

Comment:
One tiny nit: there should be another newline above this for consistency, and we can probably elide the `#` if we don't actually need a comment.